persist/src/main/resources/db/migration/V5__AuditTables.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS audit (
+    transaction_type VARCHAR(1000) NOT NULL,
```

Comment:
Why `VARCHAR(1000)`? Aren't the only possible values `insert`, `update`, `delete`?

Reply:
Could we make this an enum?

Reply:
Yes true, will make that change